internal/comby/types.go

```diff
@@ -1,16 +1,28 @@
 package comby
 
+import "archive/tar"
+
 type Input interface {
 	input()
 }
 
+type TarInput struct {
+	TarInputEventC chan TarInputEvent
+}
+
+type TarInputEvent struct {
+	Header  tar.Header
+	Content []byte
+}
+
 type ZipPath string
 type DirPath string
 type FileContent []byte
 
 func (ZipPath) input()     {}
 func (DirPath) input()     {}
 func (FileContent) input() {}
+func (TarInput) input()    {}
```

Comment:
```suggestion
func (Tar) input()    {}
```

Probably better to follow the convention here and not include `Input`